src/templates/components/Fabric.svelte

```diff
@@ -26,33 +34,74 @@
 	export let MobileLayer2BackgroundPosition: string;
 	export let MobileLayer3BackgroundImage: string;
 	export let MobileLayer3BackgroundPosition: string;
+	export let VideoURL: GAMVariable;
```

Comment:
Like what's been done with the isXL value - we can give these props a default value so that we don't have to enter them when they're not needed (eg for the Fabric XL and Fabric templates)